fakebook/src/app/notifications.service.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { TestBed } from '@angular/core/testing';
+import { Observable } from 'rxjs';
+import { Notification } from './model/notification';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+
+import { NotificationsService } from './notifications.service';
+import { HttpClientModule, HttpClient } from '@angular/common/http';
+
+
+describe('NotificationsService', () => {
+  let service: NotificationsService;
+  let httpMock: HttpTestingController;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientTestingModule,
+      ],
+      providers: [
+        NotificationsService,
+      ]
+    });
+    service = TestBed.inject(NotificationsService);
+    httpMock = TestBed.inject(HttpTestingController);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should return a list of notifications from observable', (done) => {
+
+    const testNotifications: Notification[] = [
+      { userId: 1, postId: 1, type: 'Like' },
+      { userId: 2, postId: 1, type: 'Comment' },
+      { userId: 3, postId: undefined, type: 'Follow' },
+      { userId: 4, postId: 2, type: 'Post' }
+    ];
+
+    service.Notifications$.subscribe(value => {
+      expect(value).toBe(testNotifications);
+      done();
+    });
+
+    const req = httpMock.expectOne('https://localhost:4200/notifications');
```

Comment:
4200 is probably not where you'll have that backend running, since the angular itself goes on 4200